x-pack/metricbeat/module/sql/query/query_integration_test.go

```diff
@@ -334,11 +339,52 @@ func GetOracleEnvPassword() string {
 	return password
 }
 
+// GetOracleConnectString builds the Oracle connection string with proper format
 func GetOracleConnectString(host string, port string) string {
-	time.Sleep(300 * time.Second)
 	connectString := os.Getenv("ORACLE_CONNECT_STRING")
 	if len(connectString) == 0 {
-		connectString = fmt.Sprintf("%s/%s@%s:%s/%s as sysdba", GetOracleEnvUsername(), GetOracleEnvPassword(), host, port, GetOracleEnvServiceName())
```

Comment:
This is not correct; now that there are stricter checks; moved to a format that's actually mentioned also in godror's documentation.